### DISCUSSION TITLE
When do we need to deploy Microsoft.Win32.TaskScheduler.resources.dll(s)...?

### DISCUSSION THREAD

**Discussion by sainadhakrishna** [dahall/TaskScheduler#934] [Q&A] [answered]
Helloooo Alll,

I had a small question.
I have used the TaskScheduler nuget package in my app.

When we build the application, we observe that in the bin folder (**Release or Debug**) of the corresponding project, there are locale specific resource dlls generated (**Microsoft.Win32.TaskScheduler.resources.dll**).

**Should these dlls be mandatorily deployed  along ?? or we can ignore all the corresponding locale folders.. ?** 

Where are these dlls actually useful ?? Can someone specify the API that would need these ??

I don't see any for en locale so..??

Thanks and Regards,
Krishna.

**Comment by dahall** [comment, marked as answer]
Those are local specific resource files that are automatically used by .NET based on the current locale. .NET will automatically default to "en-US" if it doesn't find a match. No API work is needed for any of that. It just comes with .NET.

You do not have to deploy those files and their sub-directories if you do not care about localization for your project. From the home page:
> The currently supported localizations include: English, Spanish, Italian, French, Chinese (Simplified), German, Polish and Russian.
